lib/experimental/Layouts/FullscreenLayout/index.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export interface StandardLayoutProps {
+  children: React.ReactNode
+}
+
+export function FullscreenLayout({ children }: StandardLayoutProps) {
+  return <div className="relative flex flex-1 [&>div]:flex-1">{children}</div>
```

Comment:
What is your thought on using `flex` as a default container across layouts? 

I thought we can have it HTML default value to keep things familiar

Reply:
In this case, what I'm trying to do is for the content to stretch. Think the `Org chart` layout, for example. Maybe it's too much and we should just let people create their own layouts in this scenario?